pipeline_tools/create_analysis_metadata.py

```diff
@@ -448,25 +459,38 @@ def main():
     parser.add_argument('--format_map',
                         required=True,
                         help='JSON file providing map of file extensions to formats.')
+    parser.add_argument('--add_md5s',
+                        help='Set to "true" to add md5 checksums to file metadata')
     args = parser.parse_args()
 
     # Get the extension_to_format mapping
     with open(args.format_map) as f:
         extension_to_format = json.load(f)
 
-    # Get outputs and add MD5 checksum to them
+    schema_url = args.schema_url.strip('/')
+
+    # Get metadata for inputs and outputs
+    inputs = get_inputs(args.inputs_file)
     with open(args.outputs_file) as f:
         output_urls = f.read().splitlines()
-    client = storage.Client()
-    output_url_to_md5 = get_md5s(output_urls, client)
+    outputs = get_outputs(output_urls=output_urls,
+                          extension_to_format=extension_to_format,
+                          schema_url=schema_url,
+                          analysis_file_version=args.analysis_file_version)
+    print('The content of outputs are: ')
+    print(outputs)
```

Comment:
nit: Move the above after `if args.add_md5s == 'true'` block so that if we enable the md5s, it will print the right outputs to the console.